shared/middleware/helpers/storage_id.rb

```diff
@@ -61,6 +61,29 @@ def storage_encrypt_app_id(storage_id, app_id)
   Base64.urlsafe_encode64(storage_encrypt("#{storage_id}:#{app_id}"))
 end
 
+def stub_storage_user_id
+  app_id = 1337 # Stub value, until unique app_id's become available.
```

Comment:
You are correct, unlike my comment. However, I still need a place to store the app_id. The "code studio projects" work should solve this for me (this hard-coded app_id is a problem on the client as well).